pkgs/by-name/ba/bant/package.nix

```diff
@@ -18,25 +18,26 @@ let
   };
 in buildBazelPackage rec {
   pname = "bant";
-  version = "0.1.5";
+  version = "0.1.6";
 
   src = fetchFromGitHub {
     owner = "hzeller";
     repo = "bant";
     rev = "v${version}";
-    hash = "sha256-3xGAznR/IHQHY1ISqmU8NxI90Pl57cdYeRDeLVh9L08=";
+    hash = "sha256-4h76ok2aN7WfD8OHIS0O2Dk924+hFXJXewKRM7XYjKw=";
   };
 
   bazelFlags = ["--registry" "file://${registry}"];
 
   postPatch = ''
     patchShebangs scripts/create-workspace-status.sh
+    rm -f .bazelversion
   '';
 
   fetchAttrs = {
     sha256 = {
       aarch64-linux = "sha256-jtItWNfl+ebQqU8VWmvLsgNYNARGxN8+CTBz2nZcBEY=";
```

Comment:
For aarch64-linux, I get the new `sha256-38O9HPKMjqpNCO+kC8hUlsJAclONVCj3oj/iVRwOEDo=`